src/help.sh

```diff
@@ -70,7 +70,29 @@ function kworkflow_man()
 
 function kworkflow_version()
 {
-  local version_path="$KW_LIB_DIR/VERSION"
+  local head_hash
+  local branch_name
+  local base_version
+  local version_path="${KW_LIB_DIR}/VERSION"
+
+  if [[ "${KW_REPO_MODE}" == 'y' ]]; then
+    # in order to run the git commands, we need to switch to the parent  directory
+    # of $KW_LIB_DIR because that is where the git repo is at.
+    cd "${KW_LIB_DIR}/.." || (complain 'Failed to find KW root directory' && return 2)
+
+    # get version info from the git repo
+    head_hash=$(git rev-parse --short HEAD)
+    branch_name=$(git rev-parse --short --abbrev-ref HEAD)
```

Comment:
Instead of `cd`, why not use the `-C` option on the git commands?

Reply:
Because the `mk_fake_git` helper in `unit/utilsh` assumes the current directory is a git directory, and runs commands in it, such as `git init`.